packages/client/src/__tests__/integration/happy/json-filtering-postgres/test.ts

```diff
@@ -208,6 +216,60 @@ describe('json-filtering(postgres)', () => {
     })
     expect(sort(b)).toMatchSnapshot()
   })
+
+  /**
+   * Filter entries that have a JSON `null` value
+   */
+  test('filter with Prisma.JsonNull', async () => {
+    const user = await prisma.user.findMany({
+      where: {
+        jsonOpt: {
+          equals: PrismaUtil.JsonNull,
+        },
+      },
+      select: {
+        email: true,
+      },
+    })
+
+    expect(user).toMatchSnapshot()
```

Comment:
Not sure a snapshot is the best here, we can keep like that and maybe discuss if we should also avoid it sometimes and prefer a simple expect length one and expect email = "email" which is stronger than a snapshot (snapshots can be updated by mistake)

Reply:
Some tests return many emails, would you prefer an inline snapshot in this case?